client/components/app.jsx

```diff
@@ -27,20 +32,54 @@ class App extends React.Component {
       .catch((error) => console.log(error)); // eslint-disable-line
   }
 
+  quickViewClick(arrayposition) {
+    const position = arrayposition;
+    this.setState({
+      quickViewVisibility: true,
```

Comment:
When you only set partial properties in state like this, does it not wipe out the other properties in state? Have you been running into any bugs where state values are getting reset unexpectedly?

Reply:
Setting partial properties is encouraged by React.  No issues losing state.